src/View/Activity/activity.html.twig

```diff
@@ -25,22 +24,38 @@
 
 		{% endfor %}
 	</div>
-
 	{# activities area #}
+	{% set counter = 0 %}
 	{% for activity in activities %}
-		<div class="activitySection row justify-content-evenly align-items-center m-5">
-			<div class="col-lg-4">
-				<h2 class="text-center m-5 essai">{{ activity.title}}</h2>
-				<p class="text-center text-primary">{{ activity.description}}</p>
+		{% if counter%2 != 0 %}
```

Comment:
non pas la peine de setter un counter à 0 ue tu incrémentes
tu as acces au tour de boucle en cours via {{ loop.index0 }} et tu peux faire un {% if loop.index0 is even }}

Reply:
au delà de ça, tu te répète beaucoup là, tu dois faire en sorte de ne pas répéter tout le bloc de code pour pair ou impair.
si tu dois juste décaler le bloc texte et le bloc image, plutot qu'en twig, fait ça en css avec un nth-chid(even) et la propriété order de flex